packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -1476,6 +1481,21 @@ class _SelectableFragment with Selectable, Diagnosticable, ChangeNotifier implem
     );
   }
 
+  @override
+  List<SelectedContentRange> getSelections() {
+    final int contentLength = range.textInside(fullText).length;
```

Comment:
You don't really need the fullText?

Reply:
Do you mean I don't need the `fullText` to calculate this? I think I can also do `range.end - range.start`.